### PR TITLE
Feat: Add window size to options

### PR DESCRIPTION
**Pull request by david-doit-intl**
Signed-off-by: david-doit-intl <david@doit-intl.com>

### CODE REVIEW

File: src/main/java/com/example/template/KafkaToBigquery.java [hunk 1]
```diff
@@ -40,64 +37,6 @@
     private static final Logger LOG = LoggerFactory.getLogger(KafkaToBigquery.class);
     private static final Gson GSON = new Gson();
 
-    public interface Options extends StreamingOptions {
```
Comment (orensham) [inline src/main/java/com/example/template/KafkaToBigquery.java:43]: @darrenbrien Agree and like it :) it is better as an encapsulated class

**Review by orensham — COMMENTED** [review]
@david-doit-intl can you please update the readme for the new window option?

File: src/main/java/com/example/template/ConsumerFactoryFn.java [hunk 2]
```diff
@@ -14,15 +14,12 @@
...
-//import jdk.internal.org.jline.utils.Log;
 
 public class ConsumerFactoryFn implements SerializableFunction<Map<String, Object>, Consumer<byte[], byte[]>> {
-    /**
-     *
-     */
     private static final long serialVersionUID = 1L;
 
     private static final Logger LOG = LoggerFactory.getLogger(ConsumerFactoryFn.class);
+    private static final Storage storage = StorageOptions.getDefaultInstance().getService();
```
[the remark this answers is not in the document]
Comment (david-doit-intl) [reply src/main/java/com/example/template/ConsumerFactoryFn.java:22]: yes, will do so.

File: src/main/java/com/example/template/ConsumerFactoryFn.java [hunk 3]
```diff
@@ -31,34 +28,24 @@ public ConsumerFactoryFn(SSLConfig sslConfiguration) {
 
     public Consumer<byte[], byte[]> apply(Map<String, Object> config) {
         if (sslConfig.isEnable) {
-            try {
```
[the remark this answers is not in the document]
Comment (david-doit-intl) [reply src/main/java/com/example/template/ConsumerFactoryFn.java:34]: no its there :)

[the remark this answers is not in the document]
Comment (orensham) [reply src/main/java/com/example/template/ConsumerFactoryFn.java:34]: Right, my mistake I missed it

**Review by orensham — APPROVED** [review]

**Pull request merged**